snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: tqdm
+version: '1.0'
+summary: A fast, extensible progress bar for Python and CLI
+description: |
+  A fast, extensible progress bar for Python and CLI
+grade: stable
+confinement: strict
+base: core18
+icon: tqdm-logo.png
+parts:
+  tqdm:
+    plugin: python
+    python-version: python2
+    source: https://github.com/tqdm/tqdm.git
+apps:
+  tqdm:
+    command: bin/tqdm
+    plugs:
+      - home
```

Comment:
is this plug required?

Reply:
Unless the app needs not access anything inside the home dir, then not. You can test and restrict further as you see fit. And to answer another question you raised earlier, there can be multiple binaries, even with different plugs if you choose so.